superset/assets/javascripts/explorev2/components/SelectField.jsx

```diff
@@ -7,12 +7,13 @@ import Select, { Creatable } from 'react-select';
 const propTypes = {
   name: PropTypes.string.isRequired,
   choices: PropTypes.array,
-  value: PropTypes.oneOfType([PropTypes.string, PropTypes.number, PropTypes.array]).isRequired,
+  value: PropTypes.oneOfType([PropTypes.string, PropTypes.number, PropTypes.array]),
```

Comment:
I'm not sure why it was there in the first place, seems like conceptually `value` can be `null`, especially if the field is `clearable`, which they all were. I'd have to run some checks on master but I'm thinking this was throwing warnings and I removed the `isRequired` to stop the warnings.